tests/test_store_invite.py

```diff
@@ -0,0 +1,54 @@
+import email.utils
```

Comment:
License please!

PyCharm has the ability to auto-add licenses when you add a new file. 👍